features/support/env.rb

```diff
@@ -74,8 +74,7 @@ def debrand_string(str)
 end
 host = "https://#{host}"
 
-$myhostname = `hostname -f`
-$myhostname.chomp!
+$myhostname = host
```

Comment:
hi @kkaempf ! thx for review! for moment i'm trying to running the suite, when i'm done i will make more order to the variables, also for the master branch :)